src/content/docs/iast/install.mdx

```diff
@@ -9,44 +9,179 @@ freshnessValidatedDate: never
+
+<CONTRIBUTOR_NOTE>
+I may need to delete the above steps as these steps do not align with our Style guide.
+</CONTRIBUTOR_NOTE>
+
 To install New Relic IAST:
 
 <Steps>
   <Step>
-    Go to <DNT>**[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST**</DNT> and click <DNT>**Set up IAST with applications**</DNT>.
+    Go to <DNT>**[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST** </DNT> and click <DNT>**Install**</DNT>.  Click <DNT>**Start trial**</DNT> to start a free 30-day trial of IAST.
+
+    <img
+      title="Start a free, 30-day IAST trial"
+      alt="Start a free, 30-day IAST trial"
+      src="/images/iast_screenshot-full_install.webp"
+    />
+
+    Click <DNT>**Set up IAST with applications**</DNT> to start trial of IAST.
 
     <img
       title="Set up IAST with applications"
       alt="Set up IAST with applications"
-      src="/images/iast_screenshot-full_install.webp"
+      src="/images/iast-instructions.webp"
     />
+
+
   </Step>
 
   <Step>
-    Confirm IAST isn't running in a production environment and click <DNT>**Continue**</DNT>.  
+    
+<Callout variant="important">
+  Before you start IAST installation, review the IAST testing steps and how exploitable vulnerabilities are detected. 
+  For more information, refer: [IAST exploitable vulnerabilities](https://docs.newrelic.com/docs/iast/exploitable-vulns/)
+</Callout>
 
-  </Step>
+## Safety check
 
+    Make sure IAST is not running in the production environment and click <DNT>**Continue**</DNT>.
+    <img
+      title="IAST Install Instruction Screen" 
+      alt="IAST Installation Instruction Screen"
+      src="/images/iast-install-steps.webp"
+    /> 
+
+  </Step>
+  
   <Step>
-    Select the language of your application and complete the steps.
 
+## Selection of application language
+
+    Select the language of your application and complete the steps. The application languages supported are: Java, Node, and Go.  
     <img
       title="Install New Relic IAST"
       alt="Install New Relic IAST"
       src="/images/iast_screenshot-crop_install.webp"
     />
-  </Step>
+
+    Once you select the application language, make sure to follow the on-screen instructions. 
+
+<Callout variant="tip">
+  - It is advisable to watch the [relevant application language video](/docs/iast/install/#check-out-these-demo-videos-for-setting-up-iast-with-different-application-languages) for successful installation of IAST.
+</Callout>    
+
+   </Step>
+
+    <Step>
+
+## Update the APM agent
+
+    <CollapserGroup>
+    <Collapser
+    id="update-apm-agent-for-java"
+    title="Update the APM agent for Java application language"
+    >
+
+Update the APM agent to the latest version. The minimum version supported is: v8.9.0. To update the java agent, follow these on-screen [instructions](https://docs.newrelic.com/docs/apm/agents/java-agent/installation/update-java-agent/) and verify the agent version using the command:
```

Comment:
Confusing. Are we updating the APM agent or the Java agent?
Capitlization for Java is wrong.